integration/ingester_limits_test.go

```diff
@@ -108,7 +114,13 @@ func TestIngesterGlobalLimits(t *testing.T) {
 
 			// Try to push as many series with the different metric name as we can.
 			for i, errs := 0, 0; i < 10000; i++ {
-				series, _, _ := generateSeries(fmt.Sprintf("test_limit_per_tenant_%d", rand.Int()), now)
+				var genSeries generateSeriesFunc
+				if i%2 == 0 {
+					genSeries = generateFloatSeries
+				} else {
+					genSeries = generateHistogramSeries
+				}
```

Comment:
This looks like something that can be handy in the future. It's already repeated twice in this PR

Reply:
It's repeated in a bunch of places. Replaced the original call where applicable.
I can add a new helper function to avoid repeating it.

Reply:
I've played around with simply generating two series per call to the generator function, but that's too much change in the semantics , would require major rewrite right away in the ingester limits test.

Reply:
I think what you have is good enough.